packages/shared/src/components/cards/v1/ActionButtons.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import dynamic from 'next/dynamic';
+import { Post, UserPostVote } from '../../../graphql/posts';
+import InteractionCounter from '../../InteractionCounter';
+import UpvoteIcon from '../../icons/Upvote';
+import CommentIcon from '../../icons/Discuss';
+import { Button, ButtonColor, ButtonVariant } from '../../buttons/ButtonV2';
+import { SimpleTooltip } from '../../tooltips/SimpleTooltip';
+import { useFeedPreviewMode } from '../../../hooks';
+import BookmarkIcon from '../../icons/Bookmark';
+import DownvoteIcon from '../../icons/Downvote';
+
+const ShareIcon = dynamic(
+  () => import(/* webpackChunkName: "shareIcon" */ '../../icons/Share'),
+);
+
+export interface ActionButtonsProps {
```

Comment:
I just realized, that the only difference here would be the layout, maybe we should import the same type used on the other ones to not bloat the amount of types/interfaces we're using. Considering it is a bit tedious sometimes to make small changes.